README.md

```diff
@@ -104,20 +105,26 @@ The event name and what's in the `data` for each event handler:
   * `event` The raw event
   * `senderId` The ID of the sender
   * `session` [A Session object](#the-session-object) you can mutate
-  * `source` One of `quickReply`, `postback`, `text`
```

Comment:
Removed as inaccurate: `message.quickReply` does not include `source` in its `data`.